src/Stratis.Bitcoin.Features.Consensus/CoinViews/Coindb/Coindb.cs

```diff
@@ -156,6 +151,8 @@ private void EnsureCoinDatabaseIntegrity()
         {
             this.logger.LogInformation("Checking coin database integrity...");
 
+            this.EndiannessFix();
```

Comment:
If this is being called here, is there any point having the call (just before this method is invoked) in the `Initialize()` method?

Reply:
Fixed.